docs/quickstart.rst

```diff
@@ -66,16 +73,17 @@ Using Contract
 
     from starknet_py.contract import Contract
     from starknet_py.net.client import Client
+    from starknet_py.net.networks import TESTNET
 
-    client = Client("testnet")
+    client = Client(TESTNET)
     key = 1234
 
     # Create contract from contract's address - Contract will download contract's ABI to know its interface.
-    contract = Contract.sync.from_address("0x01336fa7c870a7403aced14dda865b75f29113230ed84e3a661f7af70fe83e7b", client)
+    contract = await Contract.from_address("0x01336fa7c870a7403aced14dda865b75f29113230ed84e3a661f7af70fe83e7b", client)
```

Comment:
We should probably consider showing it is also possible to create Contract directly using constructor (`contract = Contract(params)`). While we support `Contract.from_address` it is not a recommended to use in production ([see description of this](https://starknetpy.readthedocs.io/en/latest/contract.html#starknet_py.contract.Contract.from_address_sync)) and it would be good to make users aware of it. 
cc @Solpatium